src/main/resources/portal.properties.EXAMPLE

```diff
@@ -106,8 +106,11 @@ ldap.attributes.last_name=sn
 ldap.attributes.given_name=givenName
 ldap.attributes.email=mail
 
+# study view settings
 # always show studies with this group
 always_show_study_group=
+mdacc.heatmap.study.meta.url=http://bioinformatics.mdanderson.org/study2url?studyid=
```

Comment:
@jjgao @zhx828 
Desire, yes probably, but no plans.  We do not have any plan to change to only https.  
This implementation was modeled after what is done for the Digital Slide Archive, 
which uses a http: URL.  (just above in the porperties file)  Does that have the same issue?